pallets/gear/src/lib.rs

```diff
@@ -1458,6 +1467,7 @@ pub mod pallet {
                 ),
             );
 
+            // Program interaction.
```

Comment:
We had a conversation on such google pattern to return something in if branches this case, but still not sure if it seems readable
@shamilsan take a look pls

Reply:
The function looks good to me.